packages/fxa-content-server/app/scripts/views/support.js

```diff
@@ -162,24 +159,28 @@ const SupportView = BaseView.extend({
     }
   },
 
-  findSubscription: function(productName) {
+  findSubscription: function (productName) {
     const subscriptions = this.model.get('subscriptions');
-    return subscriptions.find(sub => sub.product_name === productName);
+    return subscriptions.find((sub) => sub.product_name === productName);
   },
 
-  formatProductName: function(name) {
+  formatProductName: function (name) {
     const productNamePrefix = 'FxA - ';
     return `${productNamePrefix}${name}`;
   },
 
-  submitButtonUIToggle: function() {
+  unformatProductName: function (name) {
+    const productNamePrefix = /^FxA - /;
```

Comment:
With this, the `FxA - ` prefix is defined in two places; we should move the one on line 168 out to a wider scope so this function can use it.  It also needn't to be regex.  The prepending always happens so we could just strip the prefix without searching.

Reply:
sounds good!